app/datatables/teams_datatable.rb

```diff
@@ -0,0 +1,106 @@
+class TeamsDatatable < AjaxDatatablesRails::Base
+  include InputSanitizeHelper
+
+  def_delegator :@view, :link_to
+  def_delegator :@view, :team_path
+  def_delegator :@view, :leave_user_team_html_path
+
+  MEMEBERS_SORT_COL = 'members'.freeze
```

Comment:
It's only used in this datatable, so is it really worth to move it to constants.rb?